api/src/main/java/org/openmrs/module/pihcore/identifier/mexico/ConfigureMexicoIdGenerators.java

```diff
@@ -55,5 +62,52 @@ public static void configureGenerators(IdentifierSourceService identifierSourceS
         identifierSourceService.saveAutoGenerationOption(autoGenerationOption);
     }
 
+    private static void createDossierNumberGenerator(IdentifierSourceService identifierSourceService, Config config) {
+
+        PatientIdentifierType dossierIdentifierType = MetadataUtils.existing(PatientIdentifierType.class, MexicoPatientIdentifierTypes.MEXICO_DOSSIER_NUMBER.uuid());
+        String prefix = config.getDossierIdentifierPrefix();
+
+        if (prefix != null) {
+            SequentialIdentifierGenerator sequentialIdentifierGenerator = (SequentialIdentifierGenerator) Context.getService(
+                    IdentifierSourceService.class).getIdentifierSourceByUuid(
+                    CHIAPAS_DOSSIER_SOURCE_UUID);
+
+            if (sequentialIdentifierGenerator == null) {
+                sequentialIdentifierGenerator = new SequentialIdentifierGenerator();
+                sequentialIdentifierGenerator.setName("Sequential Generator for Dossier");
+                sequentialIdentifierGenerator.setUuid(CHIAPAS_DOSSIER_SOURCE_UUID);
+                sequentialIdentifierGenerator.setMaxLength(6 + prefix.length());
+                sequentialIdentifierGenerator.setMinLength(6 + prefix.length());
+                sequentialIdentifierGenerator.setPrefix(prefix);
+                sequentialIdentifierGenerator.setBaseCharacterSet("0123456789");
+                sequentialIdentifierGenerator.setFirstIdentifierBase("000001");
+                sequentialIdentifierGenerator.setIdentifierType(dossierIdentifierType);
+                identifierSourceService.saveIdentifierSource(sequentialIdentifierGenerator);
+            }
+
+            setAutoGenerationOptionsForDossierNumberGenerator(identifierSourceService, sequentialIdentifierGenerator);
+        }
+    }
+
+    // @bistenes: Is any of this actually necessary?
```

Comment:
@brandones  I'm not sure but it doesn't hurt and isn't incorrect...  :)

To be clearer, in the 1.0 admin UI this drives how this identifier works... enabling both manual entry and automatic generation of this identifier type via the old UI.  Not sure if it is used outside of that.